$NetBSD: patch-libfaad_Makefile.am,v 1.3 2022/05/15 02:35:33 dholland Exp $

Use -I instead of -iquote, which is gratuitously gcc-specific.

Separate includes from source files.

--- libfaad/Makefile.am.orig	2020-10-20 06:24:05.000000000 +0000
+++ libfaad/Makefile.am
@@ -1,6 +1,6 @@
 lib_LTLIBRARIES = libfaad.la libfaad_drm.la
 
-AM_CPPFLAGS	= -iquote $(top_srcdir)/include
+AM_CPPFLAGS	= -I$(top_srcdir)/include
 include_HEADERS = $(top_srcdir)/include/faad.h \
 		  $(top_srcdir)/include/neaacdec.h
 
@@ -15,7 +15,8 @@ libfaad_la_SOURCES = bits.c cfft.c decod
 		     pulse.c specrec.c syntax.c tns.c hcr.c huffman.c \
 		     rvlc.c ssr.c ssr_fb.c ssr_ipqf.c common.c \
 		     sbr_dct.c sbr_e_nf.c sbr_fbt.c sbr_hfadj.c sbr_hfgen.c \
-		     sbr_huff.c sbr_qmf.c sbr_syntax.c sbr_tf_grid.c sbr_dec.c \
+		     sbr_huff.c sbr_qmf.c sbr_syntax.c sbr_tf_grid.c sbr_dec.c
+libfaad_la_INCLUDES = \
 		     analysis.h bits.h cfft.h cfft_tab.h common.h \
 		     drc.h drm_dec.h error.h fixed.h filtbank.h \
 		     huffman.h ic_predict.h iq_table.h is.h kbd_win.h lt_predict.h \
